libs/blocks/global-navigation/global-navigation.js

```diff
@@ -204,9 +204,28 @@ const closeOnClickOutside = (e) => {
   }
 };
 
-const getIetfLocale = (ietfLocale) => {
-  const nonStandardLocaleMap = { no_NO: 'nb_NO' };
-  return nonStandardLocaleMap[ietfLocale] || ietfLocale;
+const getUniversalNavLocale = () => {
+  const { locale: { prefix: configPrefix = '', ietf = '' } = {} } = getConfig() || {};
+  const prefix = configPrefix.replace('/', '');
+  const DEFAULTLANG = {
+    en_US: ['ae_en', 'africa', 'au', 'be_en', 'ca', 'cis_en', 'cy_en', 'eg_en', 'gr_en', 'hk_en', 'id_en', 'ie', 'il_en', 'in', 'kw_en', 'langstore', 'lu_en', 'mena_en', 'mt', 'my_en', 'ng', 'nz', 'ph_en', 'qa_en', 'sa_en', 'sg', 'th_en', 'uk', 'vn_en', 'za'],
+    fr_FR: ['be_fr', 'ca_fr', 'ch_fr', 'lu_fr'],
+    de_DE: ['at', 'ch_de', 'lu_de'],
+    es_ES: ['ar', 'ar_es', 'cl', 'co', 'cr', 'ec', 'gt', 'la', 'mx', 'pe', 'pr'],
+  };
+  const CUSTOMLANG = {
+    be_nl: 'nl_NL',
+    ch_it: 'it_IT',
+    hk_zh: 'zh_TW',
+    no: 'nb_NO',
+    cis_ru: 'ru_RU',
```

Comment:
I'd vote we keep the keys in alphabetical order

Reply:
agree

Reply:
Agree. Made the change 👍